release-notes.txt

```diff
@@ -1,3 +1,365 @@
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release:
+===================
+
+The 2019.04 release includes a number of new features including improved
+support for ESP boards, porting of riotboot to a number of new platforms,
+802.15.4 support on the nRF52, and the ability for firmware to flash images
+into a separate boot slot. Support for several new boards and new sensors was
+added, and this release contains a number of bug fixes and test improvements.
+
+About xx pull requests with about yy commits have been merged since the last
+release and about zz issues have been solved. aa people contributed with code
+in bb days. Approximately cc files have been touched with dd insertions
+and ee deletions.
+
+
+Notations used below:
+=====================
+    + means new feature/item
+    * means modified feature/item
+    - means removed feature/item
+
+New features and changes
+========================
+
+System libraries
+----------------
+    + riotboot_flashwrite: added module to reliably write a firmware image to
+      flash
+    + crypto: added support for chacha20poly1305 AEAD symmetric key cypher
+      algorithm
+    * threads: added usage of thread_state_t enum to define thread states,
+      instead of an integer
+    * random: added usage of hwrng as seed if available
+    + app_metadata: added module to print application metadata
+    + unaligned.h: added functions to safely read values from potentially
+      unaligned pointers
+
+Networking
+----------
+    * gnrc_ipv6: adapted ping6 to original-ping-like implementation
+    * gnrc_ipv6: allowed sending empty IPv6 packets
+    - gnrc_pktbuf: removed deprecated gnrc_pktbuf_get_iovec() function
+    + NDP: allow definition of common hop limit
+    + NimBLE: added Bluetooth device scanner
+    + NimBLE: added helper functions for address printing
+    * NimBLE: updated package to the latest version
+
+Packages
+--------
+    + c25519: add support (embedded x25519 crypto library)
+
+Boards
+------
+    + Arduino nano: added support
+    + nrf52832-mdk: added support
+    + PhyTec reel board: added support
+    + MicroPython pyboard v1.1: added support
+    + Hamilton: added support
+    + saml10-xpro and saml11-xpro: added riotboot support
+    + nrf52: added riotboot support
+    + ESP32 and stm32: added command to allow runtime configuration of uart mode
+    + ESP32 and ESP8266: add support for ESP-NOW network device driver
+    * ESP8266: freed up RAM by moving most .rodata sections from DRAM to IROM
+    + Arduino: added test to verify pin mappings
+    + b-l072z-lrwan1: enabled RTT feature
+    + STM32l0: added support for dual bank flashing with OpenOCD
+    * Autonomo: changed flasher to BOSSA
+
+CPU
+---
+    + stm32l0: added riotboot support
+    + stm32f0: added support for periph_pm
+    + stm32f3 and stm32l1: added stop and standby low power modes
```

Comment:
I think there was also F7 and L4 => all STM32 families now provide periph_pm :)

Reply:
Ok, were F7 and L4 merged in this release cycle, or is this a summary of the current state, i.e. they were merged in some previous release cycle?

Reply:
I'm pretty sure it was during this release cycle, see #11211 which in the end contained #11178 and #11173

Reply:
ok thanks